js/civicrm_stripe.js

```diff
@@ -161,8 +205,9 @@
       $form = getBillingForm();
 
       // Don't handle submits generated by non-stripe processors
-      if (!$('input#stripe-pub-key').length) {
-        debugging('submit missing stripe-pub-key element');
+      if (!$('input#stripe-pub-key').length || !($('input#stripe-pub-key').val())) {
+        debugging('submit missing stripe-pub-key element or value');
+        $form.get(0).submit();
```

Comment:
Could we add a comment here as to why we're calling `$form.get(0).submit();`

Reply:
good catch - that is a mistake! I'm opening a new PR to replace this one.